Rem/Utilities/MemeModels/Renderers.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MemeGenerator;
+using MemeGenerator.Fonts;
+using SixLabors.Fonts;
+using SixLabors.ImageSharp;
+using System;
+
+namespace Rem.Utilities.MemeModels
+{
+    class TextRenderer
+    {
+        public AvailableFonts Font { get; set; }
+        public int FontSize { get; set; } = 20;
+        public string Pen { get; set; }
+        public int PenWidth { get; set; }
+        public string Brush { get; set; }
+        public string[] Alignment { get; set; }
+        public bool PreferNoScaling { get; set; }
+
+        public TextInputRenderer GetRenderer()
+            => new TextInputRenderer(
+                MemeFonts.GetFont(Font, FontSize),
+                Pen.Hex().GetPen(PenWidth),
```

Comment:
confusing unhandled NullReferenceException if the hex string isn't valid

Reply:
No it won't? Hex, GetPen, and GetBrush are extension methods, and they handle nulls. It'll just give the TextInputRenderer nulls for the pen/brush

Reply:
Ah, you're right.